health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/editVillagePopulationPopUP.js

```
@@ -47,88 +55,101 @@ const EditVillagePopulationPopUp = ({ onClose, census }) => {
             },
             workflow: {
+              ...census.workflow,
               action: workflowAction,
             },
           },
         }
-
+      },
+      {
+        onSuccess: (data) => {
+          onSuccess && onSuccess(data); // Call the onSuccess callback if provided
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)